experimental/casemapping/src/casemapping.rs

```diff
@@ -114,19 +152,73 @@ impl CaseMapping {
     /// This function only implements simple and common mappings. Full mappings,
     /// which can map one `char` to a string, are not included.
     /// For full mappings, use [`CaseMapping::to_full_uppercase`].
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use icu_casemapping::CaseMapping;
+    ///
+    /// let cm = CaseMapping::new();
+    ///
+    /// assert_eq!(cm.to_uppercase('c'), 'C');
+    /// assert_eq!(cm.to_uppercase('C'), 'C');
+    /// assert_eq!(cm.to_uppercase('ć'), 'Ć');
+    /// assert_eq!(cm.to_uppercase('γ'), 'Γ');
+    ///
+    /// assert_eq!(cm.to_uppercase('ǳ'), 'Ǳ');
+    /// ```
     pub fn to_uppercase(&self, c: char) -> char {
         self.data.get().simple_upper(c)
     }
 
     /// Returns the titlecase mapping of the given `char`.
     /// This function only implements simple and common mappings. Full mappings,
     /// which can map one `char` to a string, are not included.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use icu_casemapping::CaseMapping;
+    ///
+    /// let cm = CaseMapping::new();
+    ///
+    /// assert_eq!(cm.to_titlecase('ǳ'), 'ǲ');
+    ///
+    /// assert_eq!(cm.to_titlecase('c'), 'C');
+    /// assert_eq!(cm.to_titlecase('C'), 'C');
+    /// assert_eq!(cm.to_titlecase('ć'), 'Ć');
+    /// assert_eq!(cm.to_titlecase('γ'), 'Γ');
+    /// ```
     pub fn to_titlecase(&self, c: char) -> char {
         self.data.get().simple_title(c)
     }
 
     /// Returns the simple case folding mapping of the given char.
```

Comment:
```suggestion
    /// Returns the simple case folding of the given char.
```
We have mappings and foldings, calling the folding a mapping is going to add to already widespread confusion.